packages/kbn-test/src/functional_test_runner/lib/config/run_check_ftr_configs_cli.ts

```diff
@@ -66,24 +68,45 @@ export async function runCheckFtrConfigsCli() {
           return false;
         }
 
-        if (file.match(/mocks.ts$/)) {
+        // No FTR configs in /packages/
+        if (file.match(/\/packages\//)) {
+          return false;
+        }
+
+        if (file.match(/(mock|mocks).ts$/)) {
           return false;
         }
 
         const fileContent = readFileSync(file).toString();
 
-        if (fileContent.match(/(testRunner)|(testFiles)/)) {
+        if (
+          // explicitly define 'testRunner' or 'testFiles'
+          fileContent.match(/(testRunner)|(testFiles)/) ||
+          // export default createTestConfig
+          fileContent.match(/export\s+default\s+createTestConfig/) ||
+          // export default async function ({ readConfigFile }: FtrConfigProviderContext)
```

Comment:
Do you wanna drop these comments?

Reply:
I would keep it so the next person working on it have an idea of what text we are searching in files. Maybe rephrase it?

Reply:
Um, what do you mean? 
Sounds like I'm missing something bud.

Reply:
`fileContent` is a string with js/ts file content and we use regexp to find something that might tell if it is FTR config.
It is easy see an example that you can search repo with to find if some files still has the string, e.g.  `export default createTestConfig`

Reply:
ah, nvm I get it.